src/DogStatsd.php

```diff
@@ -155,6 +157,28 @@ public function __construct(array $config = array())
         );
 
         $this->resetTelemetry();
+
+        $originDetectionEnabled = $this->isOriginDetectionEnabled($config);
+        if ($originDetectionEnabled) {
+            $originDetection = new OriginDetection();
+            $containerID = isset($config["container_id"]) ? $config["container_id"] : "";
+            $this->containerID = $originDetection->getContainerID($containerID, $originDetectionEnabled);
+        }
+    }
+
+    private function isOriginDetectionEnabled($config)
+    {
+        if ((isset($config["origin_detection"]) && !$config["origin_detection"])) {
+            return false;
+        }
+
+        if (getenv("DD_ORIGIN_DETECTION_ENABLED")) {
+            $envVarValue = getenv("DD_ORIGIN_DETECTION_ENABLED");
+            return $envVarValue != "false";
```

Comment:
Would it make sense to match behavior of the go client in regard to [accepted](https://pkg.go.dev/strconv#ParseBool) false values?